src/Component/JsonSchema/Tests/fixtures/validator/expected/Validator/ValidationException.php

```diff
@@ -10,7 +10,7 @@ class ValidationException extends \RuntimeException
     public function __construct(ConstraintViolationListInterface $violationList)
     {
         $this->violationList = $violationList;
-        parent::__construct(sprintf('Model validation failed with %d errors.', $violationList->count()), 400);
+        parent::__construct(sprintf('Model validation failed with %d errors: ' . (string) $violationList, $violationList->count()), 400);
```

Comment:
Better change the Generator to use %s instead of string concatenation. sprintf is used here already. There is no need to concat the violationList.

Reply:
indeed. fixed.